### PR TITLE
[patch] add option for forward patching - if patch is already applied…

### PR DESCRIPTION
**Pull request by truthbk**
…, just ignore and continue.

### CODE REVIEW

File: lib/omnibus/builder.rb [hunk 1]
```diff
@@ -149,6 +149,7 @@ def patch(options = {})
       source = options.delete(:source)
       plevel = options.delete(:plevel) || 1
       target = options.delete(:target)
+      forward = options.delete(:forward) || false
```
Comment (olivielpeau) [inline lib/omnibus/builder.rb]: nitpick: could you add a description of the parameter in the method's inline docs above?

**Comment by olivielpeau** [comment]
:+1: apart from my small comment

File: lib/omnibus/builder.rb (hunk 1 above)
Comment (truthbk) [inline lib/omnibus/builder.rb]: done, thanks for noticing.

**Comment by olivielpeau** [comment]
:shipit:

**Pull request merged**